repo_resource/requirements.txt

```diff
@@ -1,2 +1,3 @@
 gitrepo==2.31.1
 ssh-agent-setup==2.0.1
+GitPython==3.1.31
```

Comment:
This seems like an additional change that has nothing to do with changing the base docker image.
Could we clarify in the commit message why the additional requirement has been added?

Reply:
Leftover from other branch, removed

Reply:
Ok